src/encoded/search.py

```diff
@@ -262,36 +262,51 @@ def collection_view(context, request):
 
 
 def build_search_types(types, doc_types):
-    """ Builds search_types based on the given doc_types
+    """
+    Builds `search_types` based on the requested search `type` in URI param (=> `doc_types`).
 
     :param types: TypesTool from the registry
-    :param doc_types: Type names we would like to search on
+    :param doc_types: Type names we would like to search on.
     :return: search_types, or a list of 'SearchResults' type candidates
     """
-    search_types = []
-    if len(doc_types) == 1:  # if we have one, add it and its base_type
-        ti = types[doc_types[0]]
-        search_types.append(ti.name + "SearchResults")
-        if hasattr(ti, 'base_types'):
+    encompassing_ti_for_all_items = None
+    if len(doc_types) < 1:
```

Comment:
Actually I don't think this is true, was another glitch (forgot to set `encompassing_ti_for_all_items = ti` under `if encompassing_ti_for_all_items is None` .. will delete + test.